source/connectors/kafka-sink-data-formats.txt

```diff
@@ -0,0 +1,241 @@
+.kafka-sink-data-formats
+
+=================================
+Kafka Sink Connector Data Formats
+=================================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Sink Data Formats
+-----------------
+
+
+Supported Data Formats
+----------------------
+
+The sink connector implementation is configurable in order to support:
+
+- **AVRO** (makes use of Confluent's Kafka Schema Registry and is the
+  recommended format)
+- **JSON with Schema** (offers JSON record structure with explicit schema
+  information)
+- **JSON plain** (offers JSON record structure without any attached schema)
+- **RAW JSON** (string only - JSON structure not managed by Kafka connect)
+
+Since key and value settings can be independently configured, it is possible
+to work with different data formats for records' keys and values respectively.
+
+.. note:: 
+   Even when using RAW JSON mode i.e. with `StringConverter
+   <https://kafka.apache.org/21/javadoc/index.html?org/apache/kafka/connect/storage/StringConverter.html>`_
+   the expected Strings have to be valid JSON.
+
+See the excellent Confluent post `Serializers Explained
+<https://www.confluent.io/blog/kafka-connect-deep-dive-converters-serialization-explained>`_
+for more information about Kafka data serialization.
+
+Supported Sink Record Structure
+-------------------------------
+
+Currently the connector is able to process Kafka Connect ``SinkRecord``
+objects with support for the following schema types `Schema.Type
```

Comment:
This link doesn't look right...
![image](https://user-images.githubusercontent.com/20050130/66350622-cf28cf80-e929-11e9-8996-505c65930280.png)